bindata/assets/config/defaultconfig.yaml

```diff
@@ -26,6 +26,7 @@ extendedArguments:
   - "-ttl" # TODO: this is excluded in kube-core, but not in #21092
   - "-bootstrapsigner"
   - "-tokencleaner"
+  - "selinux-warning-controller" # Most upstream clusters don't care about SELinux, therefore this controller is opt-in. OCP cares.
```

Comment:
@jsafrane Are you willing to add yourself/your team as the owner of this option/controller?

Reply:
Added some comment, now it's multi-line to keep it nice. PTAL

Reply:
Thanks